.github/workflows/php-test-workflow.yml

```diff
@@ -11,10 +11,6 @@ jobs:
         strategy:
             matrix:
                 include:
-                -   php: '8.1'
```

Comment:
@kedarkhaire rather than removing 8.1 check if you can skip the version for symfony 7.